_meta/beat.yml

```diff
@@ -5,7 +5,7 @@
 gpfsbeat:
   # Defines how often an event is sent to the output
   period: 60s
-  filesystem: ["/"]
+  device: ["/"]
```

Comment:
We can retain only lines beginning with a known prefix `mmrepquota` and ignore HEADERs if we already found them. Or just reset them. Which is not plausible for mmdf, apparently :) But one thing at a time.